scripts/deploy_aura_bal.py

```diff
@@ -0,0 +1,53 @@
+from brownie import (
+    accounts,
+    interface,
+    AuraBalVault,
+    AuraHandler,
+    BBUSDHandler,
+    AuraBalStrategy,
+    AuraBalZaps,
+)
+
+from tests.utils.constants import (
+    AURA_BAL_TOKEN,
+    AIRFORCE_SAFE,
+    AURA_TOKEN,
+    BBUSD_TOKEN,
+    BAL_TOKEN,
+    ADDRESS_ZERO,
+)
+
+
+def main():
+    deployer = accounts.load("mainnet-deploy")
+    vault = AuraBalVault.deploy(
+        AURA_BAL_TOKEN, {"from": deployer}
+    , publish_source=True)
+    vault.setPlatform(AIRFORCE_SAFE, {"from": deployer})
+
+    strategy = AuraBalStrategy.deploy(
+        vault, {"from": deployer}
+    , publish_source=True)
+    strategy.setApprovals({"from": deployer})
+    vault.setStrategy(strategy, {"from": deployer})
+
+    aura_handler = AuraHandler.deploy(
+        AURA_TOKEN, strategy, {"from": deployer}
+    , publish_source=True)
+    aura_handler.setApprovals({"from": deployer})
+    bbusd_handler = BBUSDHandler.deploy(
+        BBUSD_TOKEN, strategy, {"from": deployer}
+    , publish_source=True)
```

Comment:
Don't forget to add `bbusd_handler.setApprovals({"from": deployer})` in case this handler needs a `setApprovals` as well as per my other review comment on `BBUSDHandler.sol`.

Reply:
see comment below